server/eclipse-project/src/main/java/us/freeandfair/corla/csv/DominionCVRExportParser.java

```diff
@@ -498,14 +636,24 @@ private CastVoteRecord extractCVR(final CSVRecord the_line) {
     final String ballot_type =
       stripEqualQuotes(the_line.get(my_columns.get(BALLOT_TYPE_HEADER)));
```

Comment:
prefix does not need space at end, it will just create two spaces when used later on